database/migrations/2022_01_01_172746_create_runway_runway_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateRunwayRunwayTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('runway_runway', function (Blueprint $table) {
```

Comment:
I wish I could think of something to suggest as an alternative table name :D

Reply:
Haha yes... to have a meaningful table name or follow "The Laravel Way" :D